packages/@uppy/companion/package.json

```diff
@@ -30,34 +30,34 @@
   },
   "dependencies": {
     "@purest/providers": "1.0.0",
-    "@uppy/fs-tail-stream": "^1.2.0",
+    "@uppy/fs-tail-stream": "1.2.0",
```

Comment:
I think we need to pin these versions to the versions we had in `package-lock.json`, otherwise, we will effectively be downgrade some libraries to these pinned versions

Reply:
Great point! Thanks. Tried my best to pin correct versions now 👌